Monika After Story/game/script-topics.rpy

```diff
@@ -15825,3 +15833,61 @@ label monika_renewable_energy:
     m 3lksdrb "Sorry, I got carried away there, ahaha!"
     m 1tuu "Debates sure are something, huh?"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_quiet_time",
+            category=['us'],
+            prompt="Do you mind if we don't talk about anything?",
+            pool=True
+        )
+    )
+
+label monika_quiet_time:
+    if mas_isMoniNormal(higher=True):
+        m 3hub "Of course not!"
+        if persistent._mas_pm_social_personality == mas_SP_EXTROVERT:
+            m 3eka "I know silence can feel a bit awkward at times, but I don't think we should see it as a bad thing."
+            m 3lksdlb "It can be kinda hard to think of interesting things to talk about all the time, you know?"
+            m 1dka "I for one, definitely need to recharge my social batteries every once in a while." 
+
+        else:
+            m 1eka "I know I tend to talk a lot, but I totally understand if you need to have a moment of silence every now and then."
+            m 2eua "To tell you the truth,{w=0.2} I too find the occasional quiet time to be very relaxing."
+
+        m 2rubla "Although,{w=0.2} to tell you the truth...{w=0.3}{nw}"
+        extend 2hubla "simply being able to feel your presence is already very comforting to me."
+        m 2hublb "I hope you feel the same about me, ehehe~"
+        if mas_isMoniAff(higher=True):
+            m 4eka "You know, I think being able to silently hang out with each other is an important sign of a healthy relationship between two people."
+            m 4lkd "After all, if there's a need to always be talking or actively be doing something,{w=0.2} can you really say that the two people in question are truly comfortable with each other?"
```

Comment:
```suggestion
            m 4lkd "After all, can you really say you're truly comfortable with each other if there's a need to always be talking?"
```
I think it flows better like this. I suggest dropping the 'or actively be doing something' since it shows up next line.